environment.yml

```diff
@@ -2,4 +2,4 @@ channels:
     - scitools
 dependencies:
     - iris=1.*
-    - grib_api=1.14.7
+    - ecmwf_grib=1.14
```

Comment:
I would say no, minor version is being too picky.
If it works now it will work I think, as 1.16 is already out there, but we can't work with it for some tiresome backwards-incompatibility reasons (yet to be determined).
While I'm thinking of it... #57 #58